VRCSubs/vrcsubs.py

```diff
@@ -102,7 +103,7 @@ def process_sound():
         elif config["CapturedLanguage"] == "uk-UA":
             current_text = cyrtranslit.to_latin(current_text, "ua")
         else:
-            current_text = current_text
+            current_text = unidecode.unidecode(current_text)
```

Comment:
Can you wrap this in a `if not current_text.isascii():` ? 💛

Reply:
ok
but in that case also replaced the default function with `unidecode_expect_nonascii()` as per `unidecode_expect_ascii()`'s documentation:
```This is approx. five times faster if the string only contains ASCII
    characters, but slightly slower than unicode_expect_nonascii if
    non-ASCII characters are present.```